src/formatter-config.ts

```diff
@@ -51,15 +51,14 @@ export async function getConfig(
   if (configPath === LSP_CONFIG_KEY && document) {
     const clientProvider = lsp.getClientProvider();
     const client = clientProvider.getClientForDocumentUri(document.uri);
-    if (client) {
+    if (client && client.isRunning) {
       lspFormatConfig = await lsp.api.getCljFmtConfig(client);
     }
-  }
-  if (configPath === LSP_CONFIG_KEY && !lspFormatConfig) {
-    void vscode.window.showErrorMessage(
-      'Fetching formatting settings from clojure-lsp failed. Check that you are running a version of clojure-lsp that provides "cljfmt-raw" in serverInfo.',
-      'Roger that'
-    );
+    if (client.isRunning && !lspFormatConfig) {
```

Comment:
Would this error get logged even if `configPath !== LSP_CONFIG_KEY`? Does that mean this error will get logged always by default, until someone sets `configPath === LSP_CONFIG_KEY`?

Reply:
On another note, I think we need a check here for the client existing like you added in the `if` condition above -- `client && client.isRunning`. Better yet though, you can nest the check for `lspFormatConfig` in the first `if` block. Something like:

```javascript
if (client && client.isRunning) {
  lspFormatConfig = await lsp.api.getCljFmtConfig(client);
  if (!lspFormatConfig) {
    console.error(...
  }
```

Reply:
Not sure I understand the first concern. It is all protected with a check for `configPath === LSP_CONFIG_KEY`, right?

I fixed the second concern in the latest commit. Please have a look. Maybe your first concern still needs to be addressed.

Reply:
Oh, I didn't see that check before. Never mind the first concern.